sentry_sdk/integrations/gql.py

```diff
@@ -0,0 +1,90 @@
+import gql  # type: ignore[import]
+from graphql import print_ast, get_operation_ast, DocumentNode, VariableDefinitionNode  # type: ignore[import]
+from gql.transport import Transport, AsyncTransport  # type: ignore[import]
+from gql.transport.exceptions import TransportQueryError  # type: ignore[import]
+from sentry_sdk.utils import event_from_exception
+from sentry_sdk.hub import Hub
+from sentry_sdk.integrations import Integration
+from typing import Any, Dict, Optional, Tuple, Union
+
+EventDataType = Dict[str, Union[str, Tuple[VariableDefinitionNode, ...]]]
+
+
+class GQLIntegration(Integration):
+    identifier = "gql"
+
+    @staticmethod
+    def setup_once() -> None:
```

Comment:
We should have a version check here unless the SDK works with any `gql` version. See other integrations for this. You can just set this to the current `gql` release (unless you want to actually check how far back this works).